lib/ClangImporter/ImportType.cpp

```diff
@@ -211,9 +211,10 @@ namespace {
 
     ImportResult VisitType(const Type*) = delete;
 
+    // TODO: Add support for dependent types.
```

Comment:
Consider creating/referencing a bug from bugs.swift.org (for example by `// TODO(SR-12345): ...` , both in `ImportType.cpp` and in the test, so when somebody comes and fixes this they know to update these comments.

Reply:
Created `SR-13809`. I'll update the comments. Thanks.